Do not merge — trying to figure out why Circle CI is failing.

pkgw opened the pull request:
It is being super mysterious!

guillochon committed:
Update meta.yaml

guillochon committed:
MNT: Re-rendered with conda-smithy 2.3.0

guillochon committed:
MNT: Re-rendered with conda-smithy 2.3.0

guillochon committed:
bump mosfit version

guillochon committed:
had wrong hash

guillochon committed:
Bump version

guillochon committed:
Add test files to `source_files`

pkgw committed:
Various hacks to debug Circle CI failures.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

guillochon commented:
Thx @pkgw!

pkgw committed:
I bet this solves it.

The final "if" statement evaluate to false, so the exit code of the script
looks like failure. That's my guess.

pkgw commented:
Actually now I bet that my latest commit doesn't solve it. But worth a shot.

pkgw committed:
More egregious hackery.

pkgw committed:
And yet more debugging.

pkgw commented:
Investigation complete.
